shapely/constructive.py

```diff
@@ -227,7 +227,7 @@ def offset_curve(
     >>> offset_curve(line, 2)
     <LINESTRING (-2 0, -2 2)>
     >>> offset_curve(line, -2)
-    <LINESTRING (2 2, 2 0)>
+    <LINESTRING (2 0, 2 2)>
```

Comment:
noting this as a probable instance of #1436

Reply:
Good catch.  Should we normalize here to avoid this biting us later?  Or so long as the order matches whatever a particular GEOS version produces, we leave as is until it breaks?

Reply:
> Or so long as the order matches whatever a particular GEOS version produces, we leave as is until it breaks?

I would do like that for now. We should actually update that docstring to note that the exact order is GEOS-version dependent and not to be relied upon (or update our implementation to guarantee it ourselves), but that's for another PR.